dali/operators/reader/loader/nemo_asr_loader.cc

```diff
@@ -152,9 +152,11 @@ void NemoAsrLoader::ReadAudio(Tensor<CPUBackend> &audio,
 
   resample_scratch.resize(resample_scratch_sz);
 
-  DecodeAudio<OutputType, DecoderOutputType>(
+  // TODO(janton): handle offset and duration
```

Comment:
What's stopping you from doing it now?

Reply:
I am going to do it in two separate PRs, one for AudioDecoder and one for NemoAsrReader, including proper python tests